src/activities/workgroups.md

```diff
@@ -20,13 +20,13 @@ tags: page
 ----------------------------------------------------------------------------------
 
 
-[TEI Stylesheets Task Force](https://tei-c.org/activities/workgroups/tei-stylesheets-task-force-charge/)
+[TEI Stylesheets Task Force](/activities/workgroups/tei-stylesheets-task-force-charge/)
 * This group was charged in March 2022 with rewriting the current XSLT stylesheets.
```

Comment:
This has nothing to do with @joeytakeda’s changes, but I take his initiative to propose a change in this description. I would replace `This group was charged in March 2022 with rewriting the current XSLT stylesheets` with `This group was charged in March 2022 with rewriting the current set of XSLT stylesheets that tackle ODD processing for generation of schemas and documentation.` 

The description of the charges are clear,  but this summary implies that this task force is doing the rewriting of all the stylesheets.